pkg/api/clusterinfo/clusterinfo.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
Please don't pull this API in.  We should get cluster info in a more Cloud native way, without complex GRPC APIs intermediating.